src/backend/distributed/sql/udfs/create_time_partitions/latest.sql

```diff
@@ -0,0 +1,54 @@
+CREATE OR REPLACE FUNCTION pg_catalog.create_time_partitions(
+    table_name regclass,
+    partition_interval INTERVAL,
+    end_at timestamptz,
+    start_from timestamptz DEFAULT now())
+returns boolean
+LANGUAGE plpgsql
+AS $$
+DECLARE
+    -- partitioned table name
+    schema_name_text name;
+    table_name_text name;
+
+    -- record for to-be-created parttion
+    missing_partition_record record;
+
+    -- result indiciates whether any partitions were created
+    partition_created bool := false;
+BEGIN
+    IF start_from >= end_at THEN
+        RAISE 'start_from must be older than end_at';
```

Comment:
could be nice to add the values, e.g. `start_from (%) ... end_at (%)`, start_from, end_at

I was inclined to say "ahead of" or "before", but that seems confusing given the argument order. I'm still wondering a little whether should be swapping them, but I don't think that should block merging now.